### ISSUE TITLE
Zoom behavior changed in relation to Window's zoom

### ISSUE THREAD

**Issue by Casimodo72** [MicrosoftEdge/MicrosoftEdge-Documentation#58]
Hi,

I posted this info some minutes ago to an HTML5/CSS library repository here GitHub, because I originally thought that library might have a problem with Edge 38 / EdgeHTML 14 which came with the Anniversary Update.
But no, it seems to be a change in Edge's zoom behavior in relation to Window's zoom. I tried to find related information at https://developer.microsoft.com/en-us/microsoft-edge/platform/changelog. But it seems I'm not able to use your flavor of "change log" to find any related information.

Scenario: On Windows 10 tablets with high resolution, but tiny dimensions, you probably have changed Window's zoom settings via System->Display. Our settings are at 200%, not only because our app is mostly used by older folks with crappy eyes, but also because fingers of Homo Sapiens are just not that tiny and it seems that the dimensions of screens are not accessible to the OS - which is a shame or something else is wrong.

Issue: When installing the Windows 10 Anniversary Update there is a change in behavior of Edge's zoom related to Window's zoom.
Before the update: Edge's zoom was modified by Window's zoom.
After the update: Edge's zoom is not modified by Window's zoom.
This means you have to adjust Edge's zoom for your app(s) if you had a Windows zoom other than 100%.
E.g. if you had a Windows zoom of 200%. Then Edge will no longer reflect the +100% of Windows, and thus stuff will be displayed in a -100% tiny fashion.

Please, could this change be communicated more accessibly to developers?
A search for "edge browser 38 zoom behavior change anniversary update" which would be the common terms to search for, does not come up with helpful results. Neither via Google nor via Bing.

A kind tip: I you change something fundamental like that, i.e. something that could render all browser apps out there to the size of postage stamps, could you hire 1 person, who would ensure that this information finds its way into the handful of remaining search engines?

Regards
Kasimier Buchcik

**Comment by libbymc** [comment]
Hi Kasimier,

Thanks for reaching out! We did not make any changes that would affect this behavior.  This sounds like it might be a product bug.  Can you open a new issue here: [https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/)?

**Comment by Casimodo72** [comment]
Hi libbymc,
thanks for the kind reply and the good intentions to involve folks in spreading the word in order to fix issues. Think about it for a minute. No, I won't open an issue elsewhere for MS due to personal experience in the past. You can easily copy & paste this issue yourself to the audience it needs. If you don't, then it's your own issue. Excuse me for being psychological here, but we only a care a bit - not that much.

Regards
Kasimier Buchcik

**Comment by libbymc** [comment]
Hi Kasimier,
We have looked into this and have tried to reproduce it, but unfortunately we are not able to repro the behavior you are seeing.  Filing an [issue](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/) with more information would be your best option. 
Thanks!

**Comment by Casimodo72** [comment]
Yes, you just produced one of the reasons I gave up some time ago.
As already mentioned, our workaround is to increase the zoom in Edge on tablets when the Windows 10 Anniversary Update is applied. We have only ca. 30 tablets to adjust. That's not a big deal. Anyone having the same issue might find this information useful.
If you can't reproduce it, then well, I'm sorry for that.

**Comment by Casimodo72** [comment]
Excuse me for having been a bit verbose yesterday. I managed to get my pictures by sharing them to WhatsApp and saving them via WhatsApp Web to desktop. Sync/sharing photos from my phone to OneDrive does not work.
I noticed that my description of the issue was not quite correct.Edge's zoom is still being modified by Window's zoom, 
For both photos Window's zoom has been set to 200% and Edge's zoom to 200%. The

![pre-anniversary](https://cloud.githubusercontent.com/assets/10551738/17698390/46c79140-63bb-11e6-9106-a67bc5582cf8.jpg)
![post-anniversary](https://cloud.githubusercontent.com/assets/10551738/17698395/4ab5e586-63bb-11e6-98ea-dc1e9c2a9d9b.jpg)

**Comment by libbymc** [comment]
Hi Kasimier,
From the pictures, it looks like you're looking at the new tab page.  This page does have special handling which can lead to some unique behaviors. Does this issue repro on every webpage?

Also, what tablet are you using when you see this behavior?

**Comment by Casimodo72** [comment]
Hi,

1) No. I can't "repro on every webpage" (nobody can). I tried "www.welt.de" and it seems to be ok. I can repro with our SPA and the start page. I attached pictures of our SPA login page (all the other pages are too small as well). Don't ask me why one photo is in portrait and the other in landscape mode. Ask the Lumia guys instead. Again, OneDrive on my Windows 10 Lumia 930 didn't work and I had to use WhatsApp to transfer those pictures. I think such details are important if one tries to think using interdisciplinary functions.

2) Those are TrekStore's SurfTab® duo W1 3G. I have not intended to advertise those ;-) Actually they are quite limited and crappy - but cheap enough for our shitty business.

I now am able to connect with your point of view and realize that this issue is not easily reproducible. Thus I and apologize for my rants above and wish you a happy day.
As already mentioned, I'm not interested anymore in MS fixing this issue. We changed our zoom settings. Thank you very much.

(Plus, I apologize because I want folks to stop probing my email accounts, which are probed directly proportional to my MS issue posts for the last years. Could someone stop this please? I'm a funky citizen, but I don't hurt anyone physically? Is that ok enough?).

Regards
Kasimier Buchcik

![post-anniversary](https://cloud.githubusercontent.com/assets/10551738/17937076/300c07e4-6a21-11e6-9cea-3048d3429a87.jpg)
![pre-anniversary](https://cloud.githubusercontent.com/assets/10551738/17937078/32317b8a-6a21-11e6-937b-7f0ca5750535.jpg)

**Comment by libbymc** [comment]
No worries, Kasimier! We really do appreciate your feedback and extra research on this issue. I went ahead and filed an [issue ](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/8631173/)on your behalf. 
Thanks!

**Comment by BryanTheCrow** [comment]
This is related to another bug I've seen reported and has nothing to do with zoom, but rather how edge handles responsive web design... The new anniversary update took a step backwards in that it now IGNORES viewport sizing in both meta tags and in CSS, resulting in it NOT applying a given website's responsive CSS specifications and intead scaling the website to fit. Oddly enough this only applys to Edge in Tablet Mode. Edge behaves CORRECTLY in desktop mode... yet it now ignores viewport sizing in tablet mode... which makes no sense at all.

**Comment by BryanTheCrow** [comment]
Here is the bug: Upvote it and hopefully it'll get fixed...

https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/8102102/

**Comment by Casimodo72** [comment]
@BryanTheCrow: Thanks for the info!
I wish I had followed this thread just a single day longer, but I didn't, thus the mystery got resolved for me not until today.
The parent bug is marked with "by design", so we have to live with it.
[https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/7970618](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/7970618)
